src/lib/vast_elements/README.md

```diff
@@ -27,8 +27,8 @@ There are four major methods that the extending class must support
 
 1. `static selector()` – This should return a selector as a string, and is used to determine which elements your element class will consume
 
-1. `static appendElementsOnFollow()` – Returns a boolean (defaults to `true`) and controls, if following Vast's Wrapper VASTAdTagURI url's, should append or reload this Vast Elements' knowledge of the DOM. For most elements this should return true thereby appending and merging all of the visited vast tag's elements. But in a few cases such as finding a new wrapper url to follow would cause redirect loops.
+2. `static appendElementsOnFollow()` – Returns a boolean (defaults to `true`) and controls, if following Vast's Wrapper VASTAdTagURI url's, should append or reload this Vast Elements' knowledge of the DOM. For most elements this should return true thereby appending and merging all of the visited vast tag's elements. But in a few cases such as finding a new wrapper url to follow would cause redirect loops.
```

Comment:
Just FYI, some people use all `1.`s in their markdown ordered lists, because markdown renderers (like GitHub) will renumber them correctly, which means it's easier to add/remove/reorder them in the source, since you won't have to renumber them. (In fact, I think the numbers you put in are straight-up _ignored_, as long as the list starts with `1.`: https://www.markdownguide.org/basic-syntax/#ordered-lists )

Reply:
Ahh I did not know this! Thank you for pointing that out, Drew!